site/student-lectures/AIS/ais.rst

```diff
@@ -0,0 +1,163 @@
+.. _lecture:
+
+Artificial Immune System
+===============================
+
+**What is Artificial Immune Systems?**
+---------------------------------------
+   In artificial intelligence, artificial immune systems (AIS) are a
+   class of computationally intelligent, rule-based machine learning
+   systems inspired by the principles and processes of the vertebrate
+   immune system. The algorithms are typically modeled after the immune
+   system's characteristics of learning and memory for use in
+   problem-solving.
+
+   AIS emerged in the mid-1980s with articles authored by Farmer,
+   Packard and Perelson (1986) and Bersini and Varela (1990) on immune
+   networks. However, it was only in the mid-1990s that AIS became a
+   field in its own right.
+
+   Forrest et al. (on negative selection) and Kephart et al.[2]
+   published their first papers on AIS in 1994, and Dasgupta conducted
+   extensive studies on Negative Selection Algorithms.
```

Comment:
You have taken this verbatim from wikipedia.   

https://en.wikipedia.org/wiki/Artificial_immune_system

Reply:
i have added it as a reference